dashboard/config/scripts_json/vpl-csp-2021.script_json

```diff
@@ -25,7 +26,7 @@
       "version_year": "2021"
     },
     "new_name": null,
-    "family_name": null,
+    "family_name": "vpl-csp",
```

Comment:
same here, I would suggest clearing the version year instead, to be consistent with vpl-csp-2020